### PR TITLE
Core backtrace printing for all containers

### PR DESCRIPTION
**Pull request by thakurajayL**
Core backtrace printing for all containers

### CODE REVIEW

File: src/mme-app/handlers/attach_aia_ula.c [hunk 1]
```diff
@@ -149,14 +149,6 @@ process_ula_resp()
...
-    /*hardcoding apn for test*/
-    char apnn[] = "apn1";
-    char apn_enc[5] = {};
-    apn_enc[0] = strlen(apnn);
-    memcpy(&apn_enc[1], apnn, strlen(apnn));
-    ue_entry->apn.len = 5;
-
-    memcpy(&(ue_entry->apn.val), apn_enc, 5);
 
 	ue_entry->selected_apn.len = ula_msg->selected_apn.len;
```
Comment (badhrinathpa) [inline src/mme-app/handlers/attach_aia_ula.c:153]: only copying apn length and not apn ??

File: src/common/backtrace.c [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+#include <assert.h>
+#include <signal.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <err.h>
+ 
+void cause_segfault();
+void cause_crash();
+ 
+static char const * _program_name = "mme_app";
```
Comment (badhrinathpa) [inline src/common/backtrace.c]: program name seems fixed? is it updated for other apps ??

**Review by badhrinathpa — COMMENTED** [review]
Comments

File: src/mme-app/handlers/attach_aia_ula.c (hunk 1 above)
Comment (thakurajayL) [reply src/mme-app/handlers/attach_aia_ula.c:153]: memcpy is below.

File: src/common/backtrace.c (hunk 2 above)
Comment (thakurajayL) [reply src/common/backtrace.c]: Fixed.

**Review by badhrinathpa — APPROVED** [review]
+1

**Pull request merged**